reputation/src/ars.rs

```diff
@@ -114,12 +114,6 @@ where
         M: IntoIterator<Item = K>,
     {
         let identities: HashSet<K> = identities.into_iter().collect();
-        // Push activity with no entries is not allowed
-        // Note: at least the node's miner activity should be pushed
-        assert!(
```

Comment:
Maybe we can move this assert to the `update` function?

Reply:
identities is a generic parameter so it hasnt "is_empty()" function